azure_arc_servers_jumpstart/azure_stack_hci/powershell/azstack_hci_vm_deploy.ps1

```diff
@@ -0,0 +1,137 @@
+#Powershell script to deploy VM in Azure Stack HCI and Arc enable it
+
+# Environment variables for the optional configurations
+$DHCPEnabled = 'Select $true if DHCP is enabled on your environment, $false if not'
+
+    #If $DHCPEnabled = $false, please fill the following variables. Otherwise atribbute the vale $null
+    $IPAddress = 'Provide the static IP to assign the to the VM'
+    $PrefixLenght = 'Provide the lenght of the subnet mask to assing to the VM'
+    $DefaultGateway = 'Provide the default gateway to assign to the VM'
+    $DNSServer = 'Provide the DNS Server to assign to the VM'
+
+$ServerClusterEnabled = 'Select $true if you have a server cluster created, $false if the not'
+    
+    #If $ServerClusterEnabled = $true,  please provide the path to the cluster storage in the format "<Disk Letter>:\Folder"
+    #If $ServerClusterEnabled = $false, please provide the path to a folder where the VM will be created in the format "<Disk Letter>:\Folder"
+    $vmdir =  "<Disk Letter>:\Folder"
+
+# Environment variables for the VM creation
+$NodeName = 'Provide the name of the node where the VM will be created'
+$DomainName = 'Provide the name of the domain where the node is added'
+$VMName = 'Provide the name of the VM'
+$VSwitchName = 'Provide the name of the Virtual Switch'
+
+# Environment variables to onboard the VM to Azure Arc
+$subID = "Provide the subscriptionID"
+$appID =  "Provide the Service Principal ApplicationID"
+$secret = "Provide the Service Principal Secret"
+$tID = "Provide the tenantID"
+$rgroup = "Provide the Resource Group Name"
+$location = "Provide the Region"
+
+# Deployment
+
+
+$pos = $vmdir.IndexOf(":")
+$leftPart = $vmdir.Substring(0, $pos)
+$rightPart = $vmdir.Substring($pos+1)
+$nodepath = "\\" + $NodeName + '\' +  $leftPart + "$" + $rightpart
+
+# Download of Windows VM VHDx file 
+New-Item -Path $nodepath -Name "ArcJumpstart" -ItemType "directory"
+Write-Verbose "Downloading Windows Server .VHDX. This can take some time, hold tight..." -Verbose
+function downloadVHDx() {$ProgressPreference="SilentlyContinue"; Invoke-WebRequest -Uri "https://azhcicloudwitness1123.blob.core.windows.net/vhds/ArcVM-HCIJS-win.vhdx"  -OutFile $nodepath\ArcJumpstart\ArcVM-HCIJS-win.vhdx}
+downloadVHDx
+
+# Enable CredSSP
+Set-Item WSMAN:\localhost\client\auth\credssp –value $true
+Enable-WSManCredSSP -Role Client -DelegateComputer "$NodeName.$DomainName"
+
+# PS Remote Session to Host
+Write-Verbose "Insert credentials for node $NodeName ..." -Verbose
+$CustomCred = Get-Credential
+$s = New-PSSession -ComputerName "$NodeName.$DomainName" -Credential $CustomCred -Authentication Credssp 
+
+Write-Verbose "Starting Powershell Remote Session to $NodeName ..." -Verbose
+Invoke-Command -Session $s -ScriptBlock{
+
+    # Windows VM creation
+
```

Comment:
You have few spaces like when you have a blank line between the description and the code. Be consistent and remove the blank line.